arviz/plots/hpdplot.py

```diff
@@ -3,15 +3,16 @@
 from scipy.interpolate import griddata
 from scipy.signal import savgol_filter
 
-from ..stats import hpd
+from ..stats import hdi
 from .plot_utils import get_plotting_function, matplotlib_kwarg_dealiaser
 from ..rcparams import rcParams
+from ..utils import credible_interval_warning
 
 
 def plot_hpd(
```

Comment:
I think we should also change this to `plot_hpi` (with a deprecation warning)

Reply:
Added issue. Will merge this then go for that one